moped-editor/src/views/projects/projectView/ProjectComponents/ProjectComponentsList.js

```diff
@@ -54,6 +56,26 @@ const ProjectComponentsList = ({
     setIsClickedComponentRelated(false);
   };
 
+  /* Component link copy button */
```

Comment:
I stole some of John's work from the copy user email button here. Thanks John! 🙌

Reply:
thank you stack overflow!